helix-view/src/base64/LICENSE-MIT

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Rather than separate files for the note and license, can we inline this and `helix-view/src/base64/README.md` into the top of the base64 module and move that into `helix-view/src/base64.rs`?